cmd/funnel/moon.yml

```diff
@@ -0,0 +1,5 @@
+tasks:
+  start:
+    options:
+      outputStyle: "none"
```

Comment:
why? I think if anything we should control the amount of logs in the script itself via an env var